src/utils.ts

```diff
@@ -36,9 +36,8 @@ export const waitForStatus = async (
 };
 
 export const fetchPolicy = async (templatePolicy: string) => {
-  const policy = await fs.readFile(
-    path.resolve(__dirname, "..", "templates", templatePolicy),
-    "utf-8"
+  const policy = await readFile(
+    path.resolve(__dirname, "..", "templates", templatePolicy)
```

Comment:
Note that UTF-8 is already the default encoding here.

https://nodejs.org/api/fs.html#fspromisesreadfilepath-options

Reply:
I think removing the the `utf-8` changed the return type from a string to a buffer, from the link you posted:
> If no encoding is specified (using options.encoding), the data is returned as a [<Buffer>](https://nodejs.org/api/buffer.html#class-buffer) object. Otherwise, the data will be a string.

Reply:
Oof, it's a good thing you noticed that. Let me make sure.

Reply:
After restoring the encoding, and adding some debug log output to make sure I hit the codepath and was able to read the policy file, I'm more confident about the correctness of this fix.

![Screenshot 2023-05-12 at 15 10 58](https://github.com/newrelic/serverless-newrelic-lambda-layers/assets/260143/6d946843-ed3c-45ba-8492-861a40a88f7b)